base/compiler/typeinfer.jl

```diff
@@ -175,7 +175,7 @@ function store_backedges(frame::InferenceState)
     toplevel = !isa(frame.linfo.def, Method)
     if !toplevel && (frame.cached || frame.parent !== nothing)
         caller = frame.result.linfo
-        for edges in frame.stmt_edges
+        for edges in (frame.stmt_edges..., frame.src.edges)
```

Comment:
is this splat going to be a problem?
If so, what is best alternative?

Reply:
Factor it out into a method and call it on both sets of edges